Add delete account feature to remove an account from the accounts file. (#743)
foundry-rs/starknet-foundry: docs/src/appendix/cast/account/delete.md

# `delete`

> Delete an account from `accounts-file` and optionally his associated Scarb profile.

Reviewer: ```suggestion
Delete an account from `accounts-file` and optionally its associated Scarb profile.
```

Reviewer 2: I see a lot of @THenry14's suggestions are resolved but not applied, although they seem accurate. Is there any reason for it? 😅

## Required common arguments - passed by CLI or specified in Scarb.toml

* [`url`](../common.md#--url--u-rpc_url)

## `--name, -n <ACCOUNT_NAME>`
Required.

Account name which is going to be deleted.

## `--delete-profile`
Optional.

> If passed, his associated profile will be removed from Scarb.toml.

Reviewer: ```suggestion
If passed, the account's associated profile will be removed from Scarb.toml.
```

## `--network`
Optional.

> Network in `accounts-file` associated with the account.

Reviewer: ```suggestion
Network in `accounts-file` associated with the account. By default the network of rpc node.
```
